src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithEnumAndVariantWithDefaultFW.java

```diff
@@ -252,11 +252,26 @@ public Builder field2(
             int value)
         {
             assert lastFieldSet < INDEX_FIELD2 : "Field \"field2\" cannot be set out of order";
+            if (lastFieldSet < INDEX_FIELD1)
+            {
+                defaultField1();
+            }
             variantOfListRW.field((b, o, m) -> field2RW.wrap(b, o, m).set(value).build().sizeof());
             lastFieldSet = INDEX_FIELD2;
             return this;
         }
 
+        private Builder defaultField1()
+        {
+            variantOfListRW.field((b, o, m) ->
+            {
+                b.putByte(o, MISSING_FIELD_BYTE);
+                return MISSING_FIELD_BYTE_SIZE;
+            });
```

Comment:
Recommend changing this lambda to a generic `private static` method called something like `missingField`.

Note that you'll need to guarantee a change reaction to fill in the right number of missing fields if there are multiple defaulted fields when a subsequent field is set to an explicit value.